StyleGuide.md

```diff
@@ -206,124 +322,142 @@ Try to address as many of the suggestions as possible. If Vale flags a word that
+### Code
 
-### File naming
+#### Code blocks
 
-File names should match the page title. If you need to change a file name, it is necessary to do the following:
+Use separate code blocks for input and output. For example, rather than place both a SQL command and its resulting output within a single code block, place the SQL command inside one code block, followed by its output in another code block.
 
-- Add the previous page URL with the new URL to `_redirects`.
-- Replace all links to the previous file name with the new file name in the applicable docs versions.
+When you use multiple code blocks to demonstrate a task, interleave the code blocks with text that explains the content of each code block and clarifies where it stands in the overall flow.
 
-## Product names
+**Example:**
 
-All product names except CockroachDB should be written as Liquid variables unless part of front-matter, file names, or non-Markdown files. Use the following code in place of product names:
+```
+1. To view all virtual clusters on the standby, run:
 
-- **CockroachDB Cloud** : `CockroachDB {{ site.data.products.cloud }}`
-- **CockroachDB Basic** : `CockroachDB {{ site.data.products.basic }}`
-- **CockroachDB Standard** : `CockroachDB {{ site.data.products.standard }}`
-- **CockroachDB Advanced** : `CockroachDB {{ site.data.products.advanced }}`
-- **self-hosted** : `CockroachDB {{ site.data.products.core }} cluster`
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
 
-The first occurrence of a product name within a docs page should use its full name. At the writer's discretion, subsequent occurrences may be shortened to "Basic", "Advanced", or "Cloud", unless a writer (or reviewer) senses contextual ambiguity that could be improved by using the full product name. In long pages, it may be helpful to use the full name for each occurrence in a new sentence or if it's been a few paragraphs since an occurrence of the full product name.
+    The standby cluster will show the `main` virtual cluster is in a `replicating` state.
 
-## Page components
+    ~~~
+      id |  name  | data_state  | service_mode
+    -----+--------+-------------+---------------
+       1 | system | ready       | shared
+       3 | main   | replicating | none
+    (2 rows)
+    ~~~
+```
 
-### GitHub issues and pull requests
+#### Placeholders
 
-[Release notes](https://www.cockroachlabs.com/docs/releases/index.html) and [technical advisories](https://www.cockroachlabs.com/docs/advisories/index.html) contain links to individual GitHub issues and pull requests.
+Code samples often include placeholder values, to be replaced by values specific to a user's environment. To denote that a value in a code sample is a placeholder value that should be replaced, use curly brackets (`{}`).
 
-Reference issues and pull requests by their corresponding number, prepended with `#`.
+- **Example:** `SELECT * FROM TABLE {mytable};`.
```

Comment:
Suggest also adding an example from the API docs where these are commonly used in URLs, like

```
- **Example:** `--url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/nodes`
```